NSSNamespace/Initial_Bangazon_Site
# Ew order

## Changes made
- Right side navbar styling (navbar is now slimmer; it was being widened due to excess padding from the active user link.)
- If no there is no active customer, user should not have the option to purchase a product (Redundant, Fletcher already pushed up this change.  I had happened to have already made this change on this branch as well.)
- If there is an active customer, the customer's name in the navbar should link the user to the active customer's active cart.  (For the time being, this view should just display a line of test code reflecting the active user.)
- Added initial logic for cart view.

## To test
```git fetch```
```git checkout origin/ew-order```
- Make sure navbar looks pretty.
- Do not select a user.  Navigate to product detail view.  Make sure there is no option to purchase product.
- Select any customer from the dropdown, then click the active customer's name in the dropdown.  Make sure this links you to customer cart view.
- Check for any outstanding errors in ```OrderController.cs``` under the ```Cart()``` method.
